docs/OnClusterBuildInstructions.md

```diff
@@ -0,0 +1,345 @@
+# On-Cluster OS Image Builds
+
+## Prerequisites
+
+-   OpenShift Cluster w/ 4.14.
+
+-   Admin privileges on the aforementioned cluster
+
+-   An image registry that one has push and pull permissions to, which will be
+    used for storing the final OS image. Pull permissions for this image registry
+    must be added to the global pull secret. See [<u>these
+    instructions</u>](https://docs.openshift.com/container-platform/4.13/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets)
+    for how to do this. One can also use an OpenShift ImageStream within the MCO namespace for this purpose.
+
+## Initial Setup
+
+1.  Obtain the following information:
+
+    1.  _(Optional)_ Create an OpenShift ImageStream in the MCO namespace:
+        ```console
+        $ oc create imagestream os-image -n openshift-machine-config-operator
+        ```
+
+    2.  The name of a pull secret for the base OS image and the
+        extensions container. This must be created as a Secret within
+        the MCO namespace. One can clone the global pull secret, if
+        desired: (`$ oc create secret docker-registry
```

Comment:
Maybe we can put this as code block for  better formatting

Reply:
Yeah, that's a great idea!